packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

```diff
@@ -91,11 +91,11 @@ The action is `ALLOW` by default (in practice, `DENY` will never be sent to your
 
 Using the `com.okta.action.update` [command](#supported-commands) in your response, you can change the action that Okta will take.
 
-### data.context.user
-
-<ApiLifecycle access="ie" /><br>
+<!-- Need to clarify if we need to include this in the docs. right now, it has no content. in the context of the registration inline hook guide, it allows customers to include the object in the console.log().
 
+### data.context.user
 
+<ApiLifecycle access="ie" /><br> -->
```

Comment:
Hi @susanharper-okta I put the closing comment "tag" here. Without it, we have an orphan title with no content :)

Reply:
gotcha. now I see it. thanks!